conda-installation.md

```diff
@@ -0,0 +1,218 @@
+---
+highlighter: none
+layout: default
+title: Create a Portable Python Installation with Miniconda
+---
+
+The Anaconda/Miniconda distribution of Python is a common tool for 
+installing and managing Python-based software and other tools. This guide 
+describes how to use the Miniconda installer to create a conda 
+environment for use in CHTC jobs. 
+
+# Overview
+
+When should you use Miniconda as an installation method in CHTC? 
+- Your software has specific Miniconda/conda centric installation instructions
+- The above is true *and* the software has a lot of dependencies
+- You mainly use Python to do your work. 
+
+There are two ways to use Miniconda in CHTC. The first is to create your 
```

Comment:
should we say HTC instead of CHTC since HTC is more specific and CHTC would include both the HPC and HTC?